idd/Energy+.idd.in

```diff
@@ -42139,7 +42139,7 @@ ZoneHVAC:TerminalUnit:VariableRefrigerantFlow,
         \memo AirConditioner:VariableRefrigerantFlow or
         \memo AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl:* system.
         \memo Terminal units can be configured as zone, air loop or outside air system equipment.
-        \min-fields 19
+        \min-fields 30
```

Comment:
No, I don't like this. I would rather allow users to enter the minimum amount of information and the model accounts for the rest of the data (i.e., model accounts for defaults past min-fields). Is there a way to modify getObjectItem so that min-fields are filled? and all other objects that have min-fields issues are "fixed" by this change?

Reply:
@rraustad increasing min-fields is the straightforward fix for now. Users can still enter less than min-fields if desired as long as all required fields are entered, the object can be shorter than min-fields. Interfaces will likely write everything up to min-fields, but that's not a bad thing, because it documents the additional fields in the idf. Filling every default fields in `getObjectItem` may cause issues with certain objects that have `getInput` functions that rely on checking the number of fields returned from `getObjectItem`. So it's better to get this fix in and work on a more general solution separately.

Reply:
OK, you've said that before. The only required-field in ZoneHVAC:TerminalUnit:VariableRefrigerantFlow is the one just before max OAT for supp heater operation. If that field was now filled with the default I would expect diffs but I guess there is no example file that trips that limit.

    N11, \field Maximum Supply Air Temperature from Supplemental Heater
       \required-field
       \type real
       \units C
       \autosizable
       \default autosize
       \note Supply air temperature from the supplemental heater will not exceed this value.
    N12, \field Maximum Outdoor Dry-Bulb Temperature for Supplemental Heater Operation
       \type real
       \maximum 21.0
       \default 21.0
       \units C
       \note Supplemental heater will not operate when outdoor temperature exceeds this value.
    A19; \field Controlling Zone or Thermostat Location
       \type object-list
       \object-list ZoneNames
       \note Used only for AirloopHVAC equipment on a main branch and defines zone name where thermostat is located.
       \note Not required for zone equipment. Leave blank if terminal unit is used in AirLoopHVAC:OutdoorAirSystem:EquipmentList.
       \note Required when terminal unit is used on main AirloopHVAC branch and coils are not set point controlled.
       \note When terminal unit is used in air loop and is load controlled, this zone's thermostat will control operation.

Reply:
The diffs are likely triggered when the main heating coil has limited capacity and when space heating relies on supplemental heating coils for a lot. In existing example files, main heating coils are usually autosized or sized generously, so this situation doesn't occur.